packages/hooks/src/useIsP2PEnabled.ts

```diff
@@ -1,22 +1,22 @@
 import { useEffect } from 'react';
 import { useFetch, useInvalidateQuery } from '@deriv/api';
 import { useStore } from '@deriv/stores';
+import useHasP2PSupportedCurrencies from './useHasP2PSupportedCurrencies';
 
 const useIsP2PEnabled = () => {
     const { client, traders_hub } = useStore();
+    // Todo: to replace it with useAuthorize hook
     const { is_authorize, loginid, currency, is_virtual } = client;
     const { is_low_risk_cr_eu_real } = traders_hub;
     const invalidate = useInvalidateQuery();
+    // Todo: Replace it with new BE p2p_settings endpoint when its ready
     const { data, ...rest } = useFetch('website_status', { options: { enabled: is_authorize } });
 
     const is_p2p_supported_currency = Boolean(
         data?.website_status?.p2p_config?.supported_currencies.includes(currency.toLocaleLowerCase())
     );
     const is_p2p_enabled = is_p2p_supported_currency && !is_virtual && !is_low_risk_cr_eu_real;
 
-    // Todo: should replace with the next line instead once BE is fixed.
-    // const is_p2p_enabled = data?.p2p_config?.disabled === 0;
-
```

Comment:
@farhan-nurzi-deriv I believe we should still keep this todo, BE logic should be updated so we wouldn't need to have FE logic at line 15 anymore.